style.css

```diff
@@ -79,25 +79,49 @@ p {
   height: 1.25rem;
   font-family: 'Poppins', sans-serif;
   font-style: normal;
-  font-weight: 700;
-  font-size: 1.125rem;
   line-height: 1.25rem;
-  color: #6070ff;
   cursor: pointer;
   text-decoration: none;
+  font-size: 30px;
+  font-weight: 600;
+  color: #fff;
 }
 
 .menu {
   display: flex;
   align-items: center;
   justify-content: space-between;
+  cursor: pointer;
 }
 
 .menu img {
   width: auto;
   height: auto;
 }
 
+.mobile-menu {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: rgba(96, 112, 255, 0.9);
+  color: #fff;
+}
```

Comment:
- [ ] Ouch this is close but as you can see from the Figma design there's a blur effect on the background of the mobile menu so please make sure to implement it on your design as this will ensure better consistency with the original design.
- [ ] Also please make sure the cross icon is visible on your design so the user can easily close the mobile menu according to convention. ✅
- Figma design:
  ![image](https://github.com/mumbereh/my-portfolio/assets/104968236/4faf4a58-2187-4050-9fa1-653e31a8fa7d)
- Your design:
  ![image](https://github.com/katarighe/my-portfolio/assets/104968236/a60fb14e-fb48-4ba7-bc7f-8e2400039605)